google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/features/InstanceApi.java

```diff
@@ -133,8 +133,19 @@ Operation deleteAccessConfigFromNic(@PathParam("instance") String instance,
     */
    @Named("Instances:getSerialPortOutput")
    @GET
-   @Path("/{instance}/serialPort?port=4")
+   @Path("/{instance}/serialPort")
    SerialPortOutput getSerialPortOutput(@PathParam("instance") String instance);
+   
+   /**
+    * Returns the specified instance's serial port output.
+    *
+    * @param instance the instance name.
+    * @return if successful, this method returns a SerialPortOutput containing the instance's serial output.
+    */
+   @Named("Instances:getSerialPortOutput") // FIXME What does "Named" really do?!
```

Comment:
jclouds configures timeouts on the rest call invocations using @Named() annotation on the methods

See https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/rest/internal/InvokeHttpMethod.java#L67-L74
where` config.getTimeoutNanos` comes from the default InvocationConfig implementation: https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/rest/config/ReadAnnotationsAndProperties.java

A simple use-case for something like `@Named("flavor:list")` is that one can configure:

jclouds.timeouts.flavor:list = <timeout millis> // note the use of the name

to specify a particular timeout ad-hoc for that method annotated in that way.